Description: provide config file with installation paths
Author: Ludovico Cavedon <ludovico.cavedon@gmail.com>, Max Bowsher <maxb@f2s.com>
Forwarded: not-needed

Index: tortoisehg/tortoisehg/util/config.py
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ tortoisehg/tortoisehg/util/config.py	2009-11-21 01:57:20.000000000 -0800
@@ -0,0 +1,5 @@
+bin_path     = "/usr/bin"
+license_path = "/usr/share/common-licenses/GPL-2"
+locale_path  = "/usr/share/locale"
+icon_path    = "/usr/share/pixmaps/tortoisehg/icons"
+nofork       = True
