From: Andrej Shadura <andrew.shadura@collabora.co.uk>
Date: Sat, 5 Dec 2020 18:08:10 +0100
Subject: Replace the deprecated tostring() with tobytes()

Closes: #976599
---
 i18n/msgfmt.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/i18n/msgfmt.py b/i18n/msgfmt.py
index 2a817d2..4174a01 100644
--- a/i18n/msgfmt.py
+++ b/i18n/msgfmt.py
@@ -120,7 +120,7 @@ class Msgfmt:
                              7*4,               # start of key index
                              7*4+len(keys)*8,   # start of value index
                              0, keystart)       # size and offset of hash table
-        output += array.array("i", offsets).tostring()
+        output += array.array("i", offsets).tobytes()
         output += ids
         output += strs
         return output
